### PR TITLE
Add HomeHealthSummary storybook

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Some preliminary work before working on the new `HomeHealthSummary` view changes.

Add storybook story to the `HomeHealthSummary` view.
Nothing fancy, as we have already done with other views.
I have fixed some factories on the go.

Story available at `HomeHealthSummary`

### CODE REVIEW

File: assets/js/lib/test-utils/factories/executions.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { faker } from '@faker-js/faker';
 import { Factory } from 'fishery';
-import { hostFactory, resultEnum, randomObjectFactory } from '.';
+import { hostFactory, randomObjectFactory } from '.';
 
 export const checksExecutionStatusEnum = () =>
   faker.helpers.arrayElement(['running', 'completed', 'requested']);
 
+const resultEnum = () =>
```
Comment (arbulu89) [inline assets/js/lib/test-utils/factories/executions.js:9]: Need to create its own specific `resultEnum`, as the values are different.
For example, the `resultEnum` in this case doesn't include `unknown`

**Review by nelsonkopliku — APPROVED** [review]
LGTM

**Review by EMaksy — APPROVED** [review]
Hey man nice job, thanks for taking care. Besides the one comment nothing to add --> LGTM :D

File: assets/js/components/HealthSummary/HomeHealthSummary.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  },
+};
+
+export const Healthy = {
+  args: {
+    ...Random.args,
+    sapSystemsHealth: healthySummary,
+  },
+};
+
```
[the remark this answers is not in the document]
Comment (arbulu89) [reply assets/js/components/HealthSummary/HomeHealthSummary.stories.jsx:59]: I will add an `Empty` story

**Pull request merged**